Documentation/getting-started.md

```diff
@@ -0,0 +1,134 @@
+# Getting Started with acbuild
+
+The following guide will walk you through building and running a simple ACI
+with acbuild. The ACI will contain nginx, and be used to host static files.
+
+Note that some of the commands shown here must be run as root.
+
+## Making the ACI
+
+### Starting the build
+
+All acbuild builds have a context, that needs to be explicitly created when the
+build is started and deleted when the build is finished. It defaults to being
+stored in your current working directory, so if you change directories in the
```

Comment:
Newline after period, before "so if you change directories..."

Then rewrite from "so if you..." for clarity (JW:TODO: Write suggested replacement)